server/src/main/java/org/solarbank/server/ErrorMessage.java

```diff
@@ -0,0 +1,18 @@
+package org.solarbank.server;
+
+import lombok.Getter;
+
+@Getter
+public enum ErrorMessage {
+    BAD_REQUEST("Bad Request"),
+    INTERNAL_SERVER_ERROR("Internal Server Error"),
```

Comment:
Same with this. I think just be consistent and use `HttpStatus` reason phase. Then you can rename `INTERNAL_SERVER_ERROR_DETAILS` to `INTERNAL_SERVER_ERROR`.

The purpose of this enum is the error message field not the error status.

Reply:
Changed so now using HttpStatus reason phrase, and renamed `INTERNAL_SERVER_ERROR_DETAILS` to `INTERNAL_SERVER_ERROR`